### DISCUSSION TITLE
Join the worldwide Pimcore summit on November 18, 2021

### DISCUSSION THREAD

**Discussion by brusch** [pimcore/pimcore#10845] [General]
[![github-inspire-2021](https://user-images.githubusercontent.com/142037/142197149-41873ab5-f83f-4344-885b-36e7179691f4.jpg)](https://pimcore.com/inspire)

The Pimcore event of the year will unite our community once again in its seventh edition. Learn from awesome speakers and keynotes, experience product updates, and grow your network.

[**Get your free ticket NOW!**](https://pimcore.com/inspire)

**Comment by dpfaffenbauer** [comment]
![IMG_6098](https://user-images.githubusercontent.com/5981845/142207957-4dedb8a7-c7ac-415b-ae54-aad82051bbf1.jpeg)

WE ARE READY 😄  💪 🍻

**Reply by dvesh3** [reply]
Rightmost Pimconaut seems drunk 😄

**Reply by dpfaffenbauer** [reply]
outlook of what is going to happen tomorrow :P

**Comment by al3wb** [comment]
Can you please tell me where I can download the presentations for Pimcore Inpire 2021? In the chat room during the conference they promised to make the presentations publicly available.
